src/qml/qgismobileapp.qml

```diff
@@ -465,6 +465,13 @@ ApplicationWindow {
       mapSettings: mapCanvas.mapSettings
       currentLayer: dashBoard.currentLayer
       overrideLocation: gpsLinkButton.linkActive ? positionSource.projectedPosition : undefined
+      accuracyRequirementFail: {
+          if ( positioningSettings.accuracyIndicator && positioningSettings.accuracyRequirement )
+          {
+              return positioningSettings.accuracyBad > 0 && ( !positionSource.positionInfo || !positionSource.positionInfo.haccValid || !positionSource.positionInfo.hacc > badThreshold )
```

Comment:
```suggestion
              return positioningSettings.accuracyBad > 0 
              	&& ( 
              		!positionSource.positionInfo
              		|| !positionSource.positionInfo.haccValid 
              		|| !positionSource.positionInfo.hacc > badThreshold
                )
```
is the last condition ok with the `!` in front?

Reply:
That looks weird. It should just be without the `!` (i.e. if current accuracy is ">" == "less precise" than the threshold)